run_dir/static/js/lanes_ordered.js

```diff
@@ -0,0 +1,284 @@
+const vLanesOrderedMain = ({
+    data() {
+        return {
+            statistics_data: {},
+            in_focus: [null, null, null]
+        }
+    },
+    methods: {
+        fetchStatistics(key1, key2, key3) {
+            let url = '/api/v1/lanes_ordered'
+            /* Build the url, where only defined keys should be passed */
+            let my_arguments = [key1, key2, key3];
+            for (var i = 0; i < my_arguments.length; i++) {
+                let my_key = my_arguments[i];
+                let key_nr = i + 1;
+                // Break if my_key is undefined
+                var nr_of_keys = key_nr;
+                if (my_key === undefined) {
+                    var nr_of_keys = nr_of_keys - 1;
```

Comment:
Does `nr_of_keys` need to be declared again?

Reply:
No, good catch!